tests/services/logging.service.spec.ts

```diff
@@ -10,17 +11,16 @@ const getFilteredLogCount: (currentLog: string, predicate: (line: string) => boo
     .split('\n')
     .filter((line: string) => predicate(line)).length;
 const oldLogIdentifier = 'Old Log';
-const repeatOldLogStartHeader: (numberOfRepitions: number) => string = (numberofRepitions: number) => {
-  return `${loggingService.LOG_START_HEADER}\n${oldLogIdentifier}\n`.repeat(numberofRepitions);
+const repeatOldLogStartHeader: (number) => string = (numberOfRepetitions: number) => {
```

Comment:
Let's not include formatting changes that aren't related to the issue being fixed.